### ISSUE TITLE
Add election data.

### ISSUE THREAD

**Issue by meganrm** [townhallproject/THP-Pledge#8]
I was thinking a little timeline graphic would be nice: https://ant.design/components/timeline/ right under the state name.

**Comment by nathanmwilliams** [comment]
Yeah, that would definitely be cool. While we assemble the fuller election timeline data, would be cool to just have a simple Primary Date on there:

Ex> ALABAMA - Primary: June 5
